touchnet/models/kimi_audio/modeling_kimi_audio.py

```
@@ -0,0 +1,1212 @@
+        # TODO(xcsong): Make those ids configurable
+        self.kimia_text_blank = 151666   # <|im_kimia_text_blank|>
+        self.kimia_text_eos = 151667     # <|im_kimia_text_eos|>
+        self.eod_ids = [151645, 151663]  # <|im_msg_end|>, <|im_media_end|>
+
+    def sample_audio_logits(
+        self, logits: torch.Tensor, recent_tokens=None
+    ) -> torch.Tensor:
+        """Sample from audio logits with top-k, temperature and repetition penalty.
+
+        Args:
+            logits: Logits tensor of shape [batch_size, seq_len, vocab_size] or [batch_size, vocab_size]
```

Pylint (3.3.7)

[convention] Line too long (104/100)

(C0301)